### PR TITLE
prepare 6.x for docs release

### PR DESCRIPTION
**Pull request by rsill-neo4j**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-213.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-213.surge.sh

File: modules/ROOT/partials/notes.adoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# tag::note[]
+[CAUTION]
+====
+You are looking at version 6 of the GraphQL Library.
```
Comment (lidiazuin) [inline modules/ROOT/partials/notes.adoc]:
```suggestion
This is the documentation of the GraphQL Library version 6.
```

File: preview.yml [hunk 2]
```diff
@@ -52,3 +52,5 @@ asciidoc:
     experimental: ''
     copyright: 2023
     common-license-page-uri: https://neo4j.com/docs/license/
+    page-add-notes-module: ROOT
+    page-add-notes-tags: note
```
Comment (recrwplay) [inline preview.yml:56]:
```suggestion
    page-add-notes-tags: note
    neo4j-base-uri: https://neo4j.com
    neo4j-docs-base-uri: https://neo4j.com/docs
```

A suggestion for using links to other docs pages - we use these attributes in preview playbooks so we can  use the form `link:{neo4j-docs-base-uri}/...` in asciidoc and the URLs will resolve as they do in publish.yml playbooks.

File: antora.yml [hunk 3]
```diff
@@ -1,7 +1,8 @@
 name: graphql
 title: Neo4j GraphQL Library
 version: '6'
-display_version: '6-beta'
 start_page: ROOT:index.adoc
 nav:
 - modules/ROOT/content-nav.adoc
+page-add-notes-module: ROOT
+page-add-notes-tags: note
```
Comment (recrwplay) [inline antora.yml]:
```suggestion
asciidoc:
  attributes:
    page-add-notes-module: ROOT
    page-add-notes-tags: note
```

File: modules/ROOT/partials/notes.adoc [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+For the long-term support (LTS) version 5, refer to link:https://neo4j.com/docs/graphql/5/[GraphQL Library version 5 LTS].
```
Comment (recrwplay) [inline modules/ROOT/partials/notes.adoc]:
```suggestion
For the long-term support (LTS) version 5, refer to link:{neo4j-docs-base-uri}/graphql/5/[GraphQL Library version 5 LTS].
```

File: publish.yml
Comment (recrwplay) [inline publish.yml:1]: Add `  - "@neo4j-antora/antora-add-notes"` to the asciidoc extensions

**Review by recrwplay — CHANGES_REQUESTED** [review]

File: preview.yml (hunk 2 above)
Comment (lidiazuin) [reply preview.yml:56]: If I may add to the suggestion, I would say using {docs-home} instead of {neo4j-docs-base-uri} can be more practical

Comment (recrwplay) [reply preview.yml:56]: That is certainly shorter. We do use `{neo4j-docs-base-uri}/` in most repos, and it's the name provided by docs-template. But I'm not going to force absolute consistency here.

Comment (recrwplay) [reply preview.yml:56]: If `{docs-home}` then also probably `{neo4j-home}` instead of `{neo4j-base-uri}`

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-213.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-213.surge.sh

**Review by lidiazuin — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**